build/Targets/stylecop.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
+  "settings": {
+    "documentationRules": {
+      "xmlHeader": false,
+      "companyName": "Microsoft",
+      "copyrightText": "Copyright (c) {companyName}.  All Rights Reserved.  Licensed under the {licenseName}.  See License.txt in the project root for license information.",
```

Comment:
Given the current setup when I hit F5 will the debug fail to launch if the copyright is wrong? That was the #1 complaint when it came to StyleCop.

Reply:
> Given the current setup when I hit <kbd>F5</kbd> will the debug fail to launch if the copyright is wrong? 

No. The diagnostic severity is Warning, so it'll work quietly for everything except command line and CI builds where warnings are treated as errors.